.gitignore

```diff
@@ -37,3 +37,5 @@ deploy
 .vscode
 .vim/*
 */.factorypath
+.flatpak-builder
+exchange.haveno.Haveno.yaml
```

Comment:
Why ignore the new exchange.haveno.Haveno.yaml file? It's part of the source, no?

Reply:
The contents of `exchange.haveno.Haveno.yml` have to be edited during the gradle build so that the right permissions can be set, and instead of "editing it before and changing it back" or having to move the flatpak buiId to a temp folder, I decided to store it in exchange.haveno.Haveno.y**a**ml and .gitignore it. It deletes after the build succeeds, but I didn't want anyone staging it by accident if a build fails or something like that